src/screens/Signup/StepCaptcha/CaptchaWebView.tsx

```diff
@@ -26,7 +26,7 @@ export function CaptchaWebView({
   stateParam: string
   state?: SignupState
   onSuccess: (code: string) => void
-  onError: () => void
+  onError: (error: object) => void
```

Comment:
Maybe not important in this particular case but in general we should get out of the habit of assuming that an error has a particular type. E.g. you technically can `throw null` or `throw undefined` or `throw 'foo'` and it's important that this doesn't trip or break error reporting.

Reply:
Ah, yea here these do have an `object` value, since they come from `nativeEvent`. But for consistency sake going forward, sure lets switch to `unknown` here 👍

Reply:
Hmm are passing the event itself? If so, does it turn into something reasonable when we stringify it later?

Reply:
The `nativeEvent` is just an object of various values that comes back from the native code when it fires the event. In this case, `nativeEvent` has the following values (depending on where the error comes from)

<img width="567" alt="Screenshot 2024-04-29 at 10 25 04 PM" src="https://github.com/bluesky-social/social-app/assets/153161762/ca08b24b-2ab7-4dcc-99de-53106dfc1d62">
<img width="598" alt="Screenshot 2024-04-29 at 10 25 12 PM" src="https://github.com/bluesky-social/social-app/assets/153161762/5a2e1fdf-cc6d-416b-9388-04c1cd09e1ff">